frontend/src/components/professor/ActivityAssessmentDetails/ActivityAssessmentDetails.js

```diff
@@ -18,92 +18,149 @@ import {
   RemarksTextArea,
   TopInfo,
   TopInfoCard,
-  UserInfo
+  UserInfo,
+  ActivityAssesmentProfessorFileCol
 } from './ActivityAssesmentDetailsStyles'
 import { getActivityTypeName } from '../../../utils/constants'
 //https://www.flaticon.com/free-icon/user-picture_21104
 import userPicture from '../../../utils/resources/user-picture.png'
-
-//example result of GET - returns the first non-reviewed answer for activity of given Id
-// todo: remove it and use endpoint
-const exampleActivityToGrade = {
-  //answerId is an example of the answer identifier - we need to have a way to save points to a given answer somehow, not sure how it works (or will work) on back
-  answerId: 1,
-  activityType: 'task',
-  activityName: 'SuperSystem I',
-  userName: 'Jan Kowalski',
-  isLate: false,
-  activityDetails:
-    'Narysuj przykład sieci składającej się z 3 routerów, 5 komputerów, 4 switchy wraz z wypisanym rodzajem użytych kabli na każdym połączeniu.',
-  userAnswer:
-    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer ullamcorper finibus magna, a pellentesque eros consequat id. Donec vitae tristique purus, quis rutrum felis. Nam ultricies, risus at pharetra fermentum, mi sem blandit lacus, eget vulputate est orci sit amet dui. Aliquam est nunc, faucibus eget congue eu, lobortis non tortor. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aenean sodales sollicitudin ex, id viverra tellus dictum non. Etiam consectetur magna nec tortor sodales, ac dignissim eros aliquam. Integer sagittis mattis sapien. Curabitur blandit imperdiet suscipit. Phasellus placerat libero sed tristique pulvinar. Vivamus bibendum augue urna, accumsan interdum risus venenatis quis.',
-  activityPoints: 30,
-  remaining: 29
-}
+import { useEffect, useState } from 'react'
+import { useNavigate, useLocation } from 'react-router-dom'
+import ProfessorService from '../../../services/professor.service'
+import { Activity } from '../../../utils/constants'
+import Loader from '../../general/Loader/Loader'
+import { generateFullPath, PageRoutes } from '../../../routes/PageRoutes'
+import ActivityAssessmentStudentFileService from './ActivityAssessmentStudentFileService'
+import { ActivityAssessmentProfessorFileService } from './ActivityAssessmentProfessorFileService'
 
 export default function ActivityAssessmentDetails() {
-  // todo: remove it I think
-  //const navigate = useNavigate();
-  //const location = useLocation();
-  //console.log(location);
+  const navigate = useNavigate()
+  const location = useLocation()
+  const { activityId } = location.state
+
+  const [activityResponseInfo, setActivityResponseInfo] = useState(undefined)
+  const [remarks, setRemarks] = useState('')
+  const [givenPoints, setGivenPoints] = useState(0)
+  const [fileBlob, setFileBlob] = useState()
+  const [fileName, setFileName] = useState()
+
+  const resetStates = () => {
+    setRemarks('')
+    setGivenPoints(0)
+    setFileBlob(undefined)
+    setFileName(undefined)
+  }
```

Comment:
U mnie teraz gdy oceniłem jedną aktywność i przeszedłem do drugiej to nie odświeżyło tych pól i z tego co widzę to gdy podamy w ocenie tylko opis to nie czyści tego pola i pola z punktami, a gdy podamy plik a nie podamy opisu to nie czyści tego z plikiem. 

https://user-images.githubusercontent.com/63460537/185474513-d622aabc-63ed-430a-b758-8b54f6b721e3.mp4

Reply:
przy okazji zauważ że jest trochę uxowych bugów w tym :/